buffalo/algo/w2v.py

```diff
@@ -92,29 +90,29 @@ def initialize(self):
 
     def build_vocab(self):
         header = self.data.get_header()
-        self.logger.info('Caculating the frequency of words.')
-        uni = [0 for i in range(header['num_items'])]
+        self.logger.info("Caculating the frequency of words.")
+        uni = [0 for i in range(header["num_items"])]
         total_word_count = 0
         for sz in self.buf.fetch_batch():
             *_, keys = self.buf.get()
             for i in range(sz):
                 uni[keys[i]] += 1
             total_word_count += sz
-        self.logger.info(f'Reducing vocab with min_count({self.opt.min_count}).')
+        self.logger.info(f"Reducing vocab with min_count({self.opt.min_count}).")
         total_vocab = 0
-        # use = np.zeros(shape=header['num_items'], dtype=np.int32, order='F')
-        use = [0 for i in range(header['num_items'])]
-        for i in range(header['num_items']):
+        # use = np.zeros(shape=header["num_items"], dtype=np.int32, order="F")
+        use = [0 for i in range(header["num_items"])]
```

Comment:
주석은 지워도 될듯합니다. 다만 그냥 궁금한건데 `np.zeros`도 괜찮을듯 한데 여기는 왜 이리 되어있는지 궁금하긴 하네요(list comp보다 np.zeros로 0초기화 하는게 더 낫지 않나 생각이 들었습니다.

별개로 list comp시 i는 안쓰는 변수이니 그냥 `_`로 처리하면 좋을것 같습니다.

```suggestion
        use = [0 for _ in range(header["num_items"])]
```